pmm-app/pr.codecept.js

```diff
@@ -27,7 +27,8 @@ exports.config = {
     amiInstanceSetupPage: './tests/pages/amiInstanceSetupPage.js',
     pmmSettingsPage: './tests/pages/pmmSettingsPage.js',
     mysqlTableDetailsPage: './tests/pages/mysqlTableDetailsPage.js',
-    dashboardPage: './tests/pages/dashboardPage.js'
+    dashboardPage: './tests/pages/dashboardPage.js',
```

Comment:
overviewPage should already be part of dashboardPage, if not please add, just follow how we did for others.